addons/event/views/res_config_settings_views.xml

```diff
@@ -46,8 +46,12 @@
                             </setting>
                         </block>
                         <block title="Attendance" name="attendance_setting_container">
-                            <setting id="event_barcode" company_dependent="1" help="Scan badges to confirm attendances">
-                                <field name="module_event_barcode" widget="upgrade_boolean"/>
+                            <setting id="event" company_dependent="1" help="Scan badges to confirm attendances">
```

Comment:
This needs to reflect what we actually want to do. Not enable/disable scanning entirely, but enable/disable **bar** codes, as separate from QR codes.

Reply:
@thiryr do you want me to change message?

Reply:
Yes, since it doesn't actually impact scanning